src/Components/Authentication/Desktop/SignUpForm.tsx

```diff
@@ -92,14 +94,16 @@ export const SignUpForm: FormComponentType = props => {
               checked={values.acceptedTermsOfService}
               value={values.acceptedTermsOfService}
               type="checkbox"
-              name="acceptedTermsOfService"
+              name="accepted_terms_of_service"
               onChange={handleChange}
               onBlur={handleBlur}
             />
             {status && !status.success && <Error show>{status.error}</Error>}
-            <SignUpButton disabled={isSubmitting || hasErrors || !dirty}>
-              Sign Up
-            </SignUpButton>
+            <Sans size="3t">
```

Comment:
I wanted to use palette here but I'm finding I would have to duplicate it in a lot of places (i.e. sign up for mobile, login for mobile and desktop as well as forgot password). Ideally I'd like for this to be set in the `<SignUpButton>` styled div. I saw [this approach](https://github.com/artsy/reaction/blob/master/src/Styleguide/Elements/FollowIcon.tsx#L18-L22) but not sure how that might work with the way we are setting the `attr` on this element.